### DISCUSSION TITLE
can you make an support for the series x controller?

### DISCUSSION THREAD

**Discussion by SoosTieger** [cathery/sys-con#239] [Controller requests]

**Comment by doctor-amaton** [comment]
Hi!

You can currently do this using [MissionControl](https://github.com/ndeadly/MissionControl), installation is a little tricky and you have to make sure that you're using the LATEST atmosphere release.

syscon already offers support for the Series S/X controllers, so give it a try too, although, the method of connection will be through an USB C cable.

Good luck!

**Reply by SoosTieger** [reply]
OK thanks

**Reply by chrisssj2** [reply]
Syscon does not offer support for Series x controller? what are you talking about I tried it via USB-C cable. it gets shown as controller icon. But it doesnt respond to any button!

**Comment by 1stFlower** [comment]
Mission control does not support series controllers as well

**Reply by Masamune3210** [reply]
differing reasons though, mission control doesnt support Bluetooth LE yet, which is required to get the Series controllers connected
